calyx-py/calyx/latency.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# The unannotated code is in sdn.futil, and the annotated code is in sdn_static.futil.
+# The data file is sdn.data.
+code=../test/correctness/queues/fifo.futil #might have to generate .futil file
+#annotated_code=benchmarks/sdn/sdn_static.futil
+data=../test/correctness/queues/fifo.data
+# By default, the code is promoted to Piezo. To disable promotion, we pass the -d static-promotion flag.
+
+# Unannotated code, not further promoted
+# Cycle counts
+fud e -q $code --to dat --from calyx --through verilog -s verilog.data $data -s calyx.flags ' -d static-promotion ' | jq '{ "latency": .cycles }'  > results/fifo_latency.json
```

Comment:
Nice, I'm glad this has been useful! If you intend to eventually check this in, please update the comments to explain what's actually happening now!

Reply:
I actually didn't end up using it, so I will delete it!

Reply:
But hang on, you may need this (or something like this) to get latency counts!

Reply:
I think it will end up going in this repo https://github.com/cucapra/calyx-resource-eval/blob/main/sdn/scripts/sdn_four_ways_latency.sh in order to work.